templates/openwisp2/settings.py

```diff
@@ -364,3 +379,14 @@
 {{ instruction }}
 
 {% endfor %}
+
+{% if openwisp2_influxdb_install %}
```

Comment:
here the condition should be `openwisp2_monitoring`, because even if influxdb is being installed on another server, we still need to configure it